Core/include/Acts/Surfaces/detail/VerticesHelper.hpp

```diff
@@ -110,74 +74,27 @@ void createSegment(std::vector<vertex_t>& vertices,
 /// @param avgPhi The phi direction of the center if sector
 /// @param halfPhi The half phi sector if sector
 /// @param lseg The number of segments for for a full 2*pi segment
-///
 /// @return a vector of 2d-vectors
-static std::vector<Vector2D> ellispoidVertices(double innerRx, double innerRy,
-                                               double outerRx, double outerRy,
-                                               double avgPhi = 0.,
-                                               double halfPhi = M_PI,
-                                               unsigned int lseg = 1) {
-  // List of vertices counter-clockwise starting at smallest phi w.r.t center,
-  // for both inner/outer ring/segment
-  std::vector<Acts::Vector2D> rvertices;  // return vertices
-  std::vector<Acts::Vector2D> ivertices;  // inner vertices
-  std::vector<Acts::Vector2D> overtices;  // outer verices
-
-  bool innerExists = (innerRx > 0. and innerRy > 0.);
-  bool closed = std::abs(halfPhi - M_PI) < s_onSurfaceTolerance;
-
-  // Get the phi segments from the helper method
-  auto phiSegs = detail::VerticesHelper::phiSegments(
-      avgPhi - halfPhi, avgPhi + halfPhi, {avgPhi});
-
-  // The inner (if exists) and outer bow
-  for (unsigned int iseg = 0; iseg < phiSegs.size() - 1; ++iseg) {
-    int addon = (iseg == phiSegs.size() - 2 and not closed) ? 1 : 0;
-    if (innerExists) {
-      detail::VerticesHelper::createSegment<Vector2D, Transform2D>(
-          ivertices, {innerRx, innerRy}, phiSegs[iseg], phiSegs[iseg + 1], lseg,
-          addon);
-    }
-    detail::VerticesHelper::createSegment<Vector2D, Transform2D>(
-        overtices, {outerRx, outerRy}, phiSegs[iseg], phiSegs[iseg + 1], lseg,
-        addon);
-  }
+std::vector<Vector2D> ellispoidVertices(double innerRx, double innerRy,
```

Comment:
```suggestion
std::vector<Vector2D> ellipsoidVertices(double innerRx, double innerRy,
```

Reply:
Hmm, yes. That was just copied from the existing code. I'll see where it is used and rename it.

Reply:
Fixed.